ui/shared/components/panel/view-fields/TagFieldView.jsx

```diff
@@ -100,13 +101,14 @@ export const TagFieldDisplay = React.memo((
 ))
 
 TagFieldDisplay.propTypes = {
-  displayFieldValues: PropTypes.arrayOf(PropTypes.object).isRequired,
+  displayFieldValues: PropTypes.arrayOf(PropTypes.oneOfType(PropTypes.object, PropTypes.string)).isRequired,
   popup: PropTypes.func,
   tagAnnotation: PropTypes.func,
   displayAnnotationFirst: PropTypes.bool,
   displayMetadata: PropTypes.bool,
   linkTagType: PropTypes.string,
   tagLinkUrl: PropTypes.string,
+  tagLookup: PropTypes.object,
```

Comment:
did you mean to add this prop [here](https://github.com/broadinstitute/seqr/pull/4092/files#diff-274caeecd16940e867f395d6b80a130298f108e7ec0a5f2da1b0440f522326c6L187)?

Reply:
`tagLookup` is added as a prop to this `TagFieldDisplay` class on line 78, so it needs a proptype declaration here